resources/test/nested-testharness.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+/**
+ * Execute testharness.js and one or more scripts in an iframe. Report the
+ * results of the execution.
+ *
+ * @param {...function|...string} bodies - a function body. If specified as a
+ *                                         function object, it will be
+ *                                         serialized to a string using the
+ *                                         built-in
+ *                                         `Function.prototype.toString` prior
```

Comment:
The value is stringified like any other, which will probably involve `Function.prototype.toString`, but if `toString` is an own property on the function object directly, that will be used instead. This makes the documentation not strictly true, but I have to admit there's value in pointing out what the stringifier likely is.